pkg/protocol/inactivity/claim.go

```diff
@@ -0,0 +1,55 @@
+package inactivity
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+)
+
+// Claim represents an inactivity claim.
+type Claim struct {
```

Comment:
Assuming we have `tbtc.InactivityClaim` (see another comment) that represents the full claim type as defined on-chain (data + signatures), I think we should rename this one and emphasize the fact that this is just part of the claim used to build the signature. This way we will also avoid ambiguity between `tbtc.InactivityClaim` and `inactivity.Claim`. What do you think about:

```suggestion
type ClaimPreimage struct {
```

This will explicitly mark the fact that this structure holds data necessary to build the `ClaimHash` that is signed as part of the inactivity protocol.

Reply:
Done in 0e3b9e5a5e7bfe2b010ecf77509bdfe5536713a2.